tests/generators/index.html

```diff
@@ -38,6 +38,17 @@
 <script src="../../generators/php/variables.js"></script>
 <script src="../../generators/php/procedures.js"></script>
 
+<script src="../../generators/dart.js"></script>
```

Comment:
This file should be reverted, though the sort block should be added to the toolbox here.  And we also want a new unit test for the list block.  That would be in tests/generators/lists.xml.